modules/local/ensemblvep/main.nf

```diff
@@ -0,0 +1,80 @@
+process ENSEMBLVEP {
```

Comment:
IIRC we are not using the nf-core version because some of the perl plugins we use had missing dependencies. I do plan to work on this issue in the near future, so will move this module from local to nf-core in a subsequent release if/when I manage to fix it 🤞🏻